admin_guide/manage_authorization_policy.adoc

```diff
@@ -565,3 +565,59 @@ administrator, you can grant them the abilities.
 ----
 $ oadm policy add-role-to-user system:daemonset-admin <user>
 ----
+
+[[manage-authorization-policy-creating-local-role]]
+== Creating a Local Role
+
+To create a local role for a project:
+
+. Copy an existing cluster role:
```

Comment:
I think this is the wrong approach.  Copying an existing role is fine when you are testing things locally, but does not make sense when you actually intend to use that role.  The user needs to build it from scratch so that they actually understand the permissions they are about to give out.